src/org/labkey/test/pages/issues/BaseIssuePage.java

```diff
@@ -232,7 +233,7 @@ public String getComment()
 
                 for (String change : changes)
                 {
-                    Pattern pattern = Pattern.compile("(.+)\uc2bb(.*)");
+                    Pattern pattern = Pattern.compile("(.+)\\uc2bb(.*)");
```

Comment:
is there a way to do this without a regex?

Reply:
Yeah, but that's no fun :)
We could probably just split on that character (`\uc2bb` is double greater-than). Not sure why I went with the regex. Not going to worry about it for now. Nothing even calls this method